README.md

```diff
@@ -32,6 +32,7 @@ Currently, you can check if a class:
  - is not abstract
  - doc block contains a string
  - doc block not contains a string
+ - has an attribute
```

Comment:
I think that it could be useful to specify that this rule can work only for PHP > 8, what do you think?